modules/ROOT/pages/db-util.adoc

```diff
@@ -255,13 +126,13 @@ You can also split your DB structure among several CSV files. In this case, incl
 </dbserver:config>
 ----
 
-This creates two tables `FILE_NAME` and `FILE_NAME_1`
+This creates two tables `FILE_NAME` and `FILE_NAME_1`.
 
-== MUnit DB server Processors
+== MUnit DB Server Processors
 
-The MUnit DB server also offers a few other features, outlined in this section.
+The DB util has two processors to interact with your DB server:
```

Comment:
Should we be referring to "database utilities" or "DB utilities" rather than "DB util"? Or should DB util be `DB util`?